src/server.h

```diff
@@ -244,8 +244,13 @@ class Server : public con::PeerHandler, public MapEventReceiver,
 			{ return m_shutdown_requested; }
 
 	// request server to shutdown
-	inline void requestShutdown(void)
-			{ m_shutdown_requested = true; }
+	inline void requestShutdown() { m_shutdown_requested = true; }
```

Comment:
Why are m_shutdown_msg and m_shutdown_ask_reconnect left unchanged when using this version of the function?